assets/components/pages/event-detail/event-detail.twig

```diff
@@ -1,116 +1,162 @@
-{% set isHomepage = true %}
+{% set isSpecial = true %}
 
 {% extends "@templates/base/base.twig" %}
 
 {% block header %}
```

Comment:
The header part of this page is not very elegant... Maybe a discussion with our fellow designers can be a good idea 😉 

<img width="713" alt="capture d ecran 2018-08-24 a 09 47 57" src="https://user-images.githubusercontent.com/2025924/44572026-dc651800-a782-11e8-9ab2-8c5f4f7e81b6.png">

Reply:
You read my mind https://github.com/epfl-idevelop/elements/issues/217#issuecomment-415316089